CHANGELOG.md

```diff
@@ -7,11 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
-* None
+- Removed `try_` prefix from all trait methods.
 
 ## [0.1.0] - 2021-05-18
 
 Initial release to crates.io.
 
-[Unreleased]: https://github.com/rust-embedded-community/embedded-storage/compare/v0.1.0...HEAD
-[0.1.0]: https://github.com/rust-embedded-community/embedded-storage/releases/tag/v0.1.0
\ No newline at end of file
+[unreleased]: https://github.com/rust-embedded-community/embedded-storage/compare/v0.1.0...HEAD
```

Comment:
```suggestion
[Unreleased]: https://github.com/rust-embedded-community/embedded-storage/compare/v0.1.0...HEAD
```

Reply:
Wow I have no idea what caused  this to be changed...